src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUES_OF_SIDE = 20;
+    private static final int MAX_RADIUS = 15;
+    private static final int UNIQUE_RADIUS = 10;
+
+    private ColorSupplier color = new ColorSupplier();
+    private Random random = new Random();
+    private Figure circleUnique = new Circle(UNIQUE_RADIUS, Color.white.name());
+
+    private Figure[] figures = {
+            new Rectangle(randomNumber(), randomNumber(), color.getRandomColor()),
+            new Circle(random.nextInt(MAX_RADIUS), color.getRandomColor()),
```

Comment:
The problem with this approach is we will always have the same circle, rectangle etc for one program run, we want to have a different figure each time getRandomFigure method is called